pkg/converter/sloppy_file_test.go

```diff
@@ -0,0 +1,86 @@
+package converter_test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/ghodss/yaml"
+	"github.com/google/go-cmp/cmp"
+	"github.com/sloppyio/sloppose/pkg/converter"
+)
+
+var expectedSloppyYml = `project: pkg
+services:
+  apps:
+    busy_env:
+      cmd: sleep 20
+      dependencies:
+      - ../apps/db
+      - ../apps/wordpress
```

Comment:
I don't think this dependency should be here (only ../apps/db) - sloppy.io dependencies are implemented by Marathon, which are transitive: https://mesosphere.github.io/marathon/docs/rest-api.html#dependencies-array-of-strings